src/pages/AddForm.js

```diff
@@ -0,0 +1,232 @@
+import styled from 'styled-components/macro';
+import { NavLink } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+
+export default function AddForm({ handleMyIssues }) {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      user: '',
+      title: '',
+      body: '',
+      milestone: '',
+      labels: '',
+      isPinned: false,
+    },
+  });
+  const user = watch('user');
+  const title = watch('title');
+  const body = watch('body');
+  const milestone = watch('milestone');
+  const labels = watch('labels');
+  const maxTitleLength = 100;
+  const maxUserLength = 30;
+  const maxBodyLength = 1000;
+  const maxMilestoneLength = 50;
+  const maxLabelsLength = 100;
+
+  const separatedLabels = labels
+    .split(',')
+    .map(label => label.trim())
+    .filter(tag => tag.length > 0);
+
+  return (
+    <Container
+      onSubmit={handleSubmit(data => onSubmit(data))}
+      autoComplete="off"
+    >
+      <Navlink to="/">
+        <svg
+          width="38"
+          height="38"
+          fill="#0085dc"
+          viewBox="0 0 16 16"
+          title="back"
+        >
+          <path d="M8 0a8 8 0 1 0 0 16A8 8 0 0 0 8 0zm3.5 7.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z" />
+        </svg>
+      </Navlink>
+      <small>Alle Felder mit Sternchen* sind Pflichtfelder.</small>
+      <FlexContainer>
+        <Label htmlFor="user">User*:</Label>
+        <Counter>{maxUserLength - user.length}</Counter>
+      </FlexContainer>
+      <InputField
+        {...register('user', {
+          minLength: { value: 2, message: 'Name is to short' },
```

Comment:
maybe you could add a maxLength here too to throw a error message to the user when he reach the maximum